libifupdown/state.c

```diff
@@ -29,8 +29,13 @@ lif_state_read(struct lif_dict *state, FILE *fd)
 		char *bufp = linebuf;
 		char *ifname = lif_next_token(&bufp);
 		char *refcount = lif_next_token(&bufp);
+		char *explicit = lif_next_token(&bufp);
 		size_t rc = 1;
 		char *equals_p = strchr(linebuf, '=');
+		bool is_explicit = false;
+
+		if (*explicit)
```

Comment:
Shouldn't we check for the keyword here?

Reply:
I suppose we should, in case we extend the state file again.

Reply:
Done in acaeef8.